source/kraken/worker.cpp

```diff
@@ -772,6 +752,31 @@ void Worker::journeys(const pbnavitia::JourneysRequest& request, pbnavitia::API
     }
 }
 
+static std::pair<const type::EntryPoint, const boost::optional<const type::EntryPoints&>> get_center_and_stop_points(
+    const JourneysArg& arg) {
+    // If we have a center
+    // Origins or destinations are already computed stop_points
```

Comment:
This is why I'd rather have a `is_street_network_allowed_at_center` parameter: here you suppose some behavior from the caller but it's implicit and undocumented.
As the caller is quite far (python component, and boiler-platted few times in C++) I think it's better to have that param.

And it will also more clearly allow fancy uses in the future.

Reply:
i agree with that for the sake of readability

Reply:
I'll add that in a new PR.